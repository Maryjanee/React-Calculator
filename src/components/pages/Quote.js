const Quote = () => (
  <div className="container">
    <p>Mathematics is not about numbers, equations, computations, or algorithms:</p>
    <p>it is about understanding. -William Paul Thurston</p>
  </div>
);

export default Quote;
