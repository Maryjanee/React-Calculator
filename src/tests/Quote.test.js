import TestRenderer from 'react-test-renderer';
import Quote from '../components/pages/Quote';

it('Matches Snapshot for Quote component', () => {
    const QuoteShot = TestRenderer
    .create(<Quote />)
    .toJSON();
  expect(QuoteShot).toMatchSnapshot();
}) 