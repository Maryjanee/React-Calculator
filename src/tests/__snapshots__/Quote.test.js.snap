// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Snapshot for Quote component 1`] = `
<div
  className="container"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms:
  </p>
  <p>
    it is about understanding. -William Paul Thurston
  </p>
</div>
`;
